code/ensemble/utils/create-all-methods-ensembles.R

```diff
@@ -15,21 +15,59 @@ all_dates <- vroom(here("code", "ensemble", "EuroCOVIDhub",
 # Get all methods
 all_methods <- sub("^.*-", "", dir(here("ensembles", "data-processed")))
 
-# Run ensembles over all methods and past forecast dates
+# Run all ensembles for all past dates ------------------------------------
 ensembles <- run_multiple_ensembles(forecast_dates = all_dates,
                                     methods = all_methods,
-                                    exclude_models = exclude_by_date)
+                                    exclude_models = exclude_by_date) %>%
+  transpose() %>%
+  simplify_all()
+results <- ensembles$result %>%
+  compact()
 
 # Save in code/ensemble/forecasts/model directory as forecast_date.csv
 res <- lapply(all_methods, function(x)
   suppressWarnings(dir.create(here("ensembles", "data-processed",
                                    paste0("EuroCOVIDhub-", x)),
                               recursive = TRUE)))
 
-walk(ensembles,
+walk(results,
      ~ vroom_write(x = .x$ensemble,
                    path = here("ensembles", "data-processed",
                                paste0("EuroCOVIDhub-", .x$method),
                                paste0(.x$forecast_date, "-EuroCOVIDhub-",
                                       .x$method, ".csv")),
                    delim = ","))
+
+
+# Run one ensemble over all past dates ---------------------------
+# Define method
+single_method <- "relative_skill"
+
+# Run
+ensembles <- run_multiple_ensembles(forecast_dates = all_dates,
```

Comment:
Not sure what the intention here is - doesn't this repeat what was already done above when running all methods?

Reply:
Ah - yes, it does repeat the above (except for saving weights), I was just using for testing/debugging. Removed.